src/Geometry.jl

```diff
@@ -315,15 +315,18 @@ end
 # This object cannot implement the Triangulation interface in a strict sense
 """
 """
-struct DistributedTriangulation{Dc,Dp,A,B} <: GridapType
+struct DistributedTriangulation{Dc,Dp,A,B,C} <: GridapType
   trians::A
   model::B
+  strategy::C
```

Comment:
A comment about Julia that does not imply any action on your side, with the aim it might be helpful in the future (if you don't know it already). 

Note that Julia does not force you to declare a member variable of type `C`. You may have the template parameter `C` without declaring `strategy` in this particular case. I am aware that we need to access `strategy` in this case, in particular, in the `InterfaceTriangulation` constructor.